### PR TITLE
Follow-up PR for Linux-arm; CLI:master

### PR DESCRIPTION
**Pull request by ghost**
- The badge for 'Linux-arm' should be: "linux_arm_Release_version_badge.svg" not "linux_x64_Release_version_badge.svg"
- Add comment for the LZMA non-inclusion for 'arm*': https://github.com/dotnet/cli/issues/8800
- Turn off collection of test results in Jenkins for 'arm*' since the leg is build-only

Follow-up to PR: https://github.com/dotnet/cli/pull/8791

### CODE REVIEW

**Comment by Petermarcu** [comment]
Does this take care of making sure the first run isn't run every time and telemetry is still enabled? That could be in a follow up but I wanted to understand where this got us.

**Comment by johnbeisner** [comment]
@Petermarcu 
"Does this take care of making sure the first run isn't run every time..." - it appears to me that something was run the first time through - but extraction of the LZMA was not a piece of that first-time experience.

FYI: The corrected arm-SDK is still on your device.

**Comment by livarcocc** [comment]
@johnbeisner we need to validate that when the lzma is not found we will still right the sentinel file and not keeping trying to run first run for every execution of the CLI, therefore showing the first run message to the user all the time as well. Can you follow up on that?

**Comment by johnbeisner** [comment]
@Petermarcu 
@livarcocc 
This appears to be the sentinel file from our testing: 
"./.dotnet/2.1.300-preview2-008360.dotnetFirstUseSentinel"

**Comment by johnbeisner** [comment]
@Petermarcu
@livarcocc
There are three messages displayed on the first invocation that are not there on the second invocation: 
**Welcome to .NET Core!
Telemetry
ASP.NET Core**

And there are 2 added sentinel files:
./.dotnet/2.1.300-preview2-008360.aspNetCertificateSentinel
./.dotnet/2.1.300-preview2-008360.dotnetFirstUseSentinel

**Comment by Petermarcu** [comment]
That looks good to me.

**Comment by johnbeisner** [comment]
[LinuxArmDotnetSecondInvocation.txt](https://github.com/dotnet/cli/files/1816442/LinuxArmDotnetSecondInvocation.txt)
[LinuxArmDotnetFirstInvocation.txt](https://github.com/dotnet/cli/files/1816443/LinuxArmDotnetFirstInvocation.txt)

**Comment by Petermarcu** [comment]
@livarcocc are you good with this change now? The output looks right to me.

**Comment by johnbeisner** [comment]
@dotnet/dotnet-cli 
Can I get an approval on this PR please.
Note: the 'Linux arm Debug Build' CI leg should be repaired after this merge...

**Review by peterhuene — APPROVED** [review]

**Pull request merged**
